api/app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,13 @@
+module Types
+  class UserType < Types::BaseObject
+    graphql_name "User"
+
+    implements GraphQL::Types::Relay::Node
+
+    field :firstName, String, null: false, hash_key: :first_name
+
+    field :lastName, String, null: false, hash_key: :last_name
+
+    field :email, String, null: false
```

Comment:
Is there a more elegant way to choose the fields included from the model here, almost like lodash `_pick()`?

Reply:
By convention Ruby uses underscore case, use underscore_case on these fields and graphql ruby will convert that to the graphql converntion of snakeCase

field :last_name....

Reply:
oh that's cool - I didn't realize ruby graphql would convert snake_case to camelCase for these.